src/SFA.DAS.EmployerIncentives.Domain/ValueObjects/CollectionCalendar.cs

```diff
@@ -93,6 +93,13 @@ public DateTime GetAcademicYearEndDate(string academicYearId)
             return academicYear.EndDate;
         }
 
+        public DateTime GetCensusDate()
```

Comment:
Can the caller not use the existing method to get the active period and read the property of the object?  Rather than have a whole load of specific functions just use the data already on the object.

Reply:
Good point. I originally thought the census date method would be more complex than that, so split it out.
That's removed now